assets/js/base/components/product-price/index.tsx

```diff
@@ -183,25 +183,25 @@ export interface ProductPriceProps {
 	 * Applies the `wc-block-components-product-price--align-${ align }` utility
 	 * class to the wrapper.
 	 */
-	align?: 'left' | 'center' | 'right';
+	align?: 'left' | 'center' | 'right' | undefined;
```

Comment:
This part had been changed to prevent a TS error in the following sections:
- https://github.com/woocommerce/woocommerce-blocks/blob/8e8b4a003d0d95c98b0cb388d034a0b73709f1e2/assets/js/atomic/blocks/product-elements/price/block.tsx#L46
- https://github.com/woocommerce/woocommerce-blocks/blob/8e8b4a003d0d95c98b0cb388d034a0b73709f1e2/assets/js/atomic/blocks/product-elements/price/block.tsx#L59-L76